src/pages/Admin/Room/Modal/Item/Item.tsx

```diff
@@ -97,7 +99,7 @@ const RoomModalItem: React.FC<RoomModalItemProps> = ({
     }
   }, [editValues, onSubmitHandler, template, useUrl, user, values, venueId]);
 
-  const handleOnChange = (val: string) => setValue("image_url", val, false);
+  const handleOnChange = (val: string) => setValue("image_url", val);
```

Comment:
What does the `false` here do? It's hard to tell given we aren't using named parameters on `setValue`

Reply:
The third parameter was `triggerValidation` and it was set on `false`. And when you upload a new image the validation was not triggering.

Reply:
Makes sense.